src/debugger/mock-ui/UISession.js

```diff
@@ -90,8 +101,20 @@ export class UISession {
   }
 
   _processEvent(event: DebugProtocol.Event) {
-    // to be implemented
-    console.log(event);
+    if (event.event === "output") {
+      console.log("Prepack output:", event.body.output);
```

Comment:
I would suggest you provide a single output function instead of direct call console.log() here. Because one day we may want to change the mock UI to write to a file or other output source. At that time, modify a single function is much pleasant than searching and replace all the console.log.